packages/cli/src/lib/index.js

```diff
@@ -120,6 +123,17 @@ export const getPaths = () => {
   }
 }
 
+/**
+ * This returns the config present in `prettier.config.js` of a Redwood project.
+ */
+export const prettierOptions = () => {
+  try {
+    return require(path.join(getPaths().base, 'prettier.config.js'))
```

Comment:
I know people are no suppose to change prettier config name but if they do this will stop working. File names such as `.prettierrc.js` is also [accepted](https://prettier.io/docs/en/configuration.html#basic-configuration) by prettier. 

Prettier itself is using [cosmiconfig](https://github.com/davidtheclark/cosmiconfig) to load the config file. We can also use that package here for making it more robust.

Reply:
That's awesome! I'm inclined to open a new issue for adding support for this.